content/zh/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,690 @@
+the required group permissions for root(0) group.
+
+Exit your shell:
+-->
+你会看到 `gid` 值为 3000，也就是 `runAsGroup` 字段的值。
+如果 `runAsGroup` 被忽略，则 `gid` 会取值 0（root），而进程就能够与 root
+用户组所拥有以及要求 root 用户组访问权限的文件交互。
+
+退出你的 Shell：
+
+```shell
+exit
+```
+
+<!--
+## Configure volume permission and ownership change policy for Pods
+-->
+## 为 Pod 配置卷访问权限和属主变更策略
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+<!--
+By default, Kubernetes recursively changes ownership and permissions for the contents of each
+volume to match the `fsGroup` specified in a Pod's `securityContext` when that volume is
+mounted.
+For large volumes, checking and changing ownership and permissions can take a lot of time,
+slowing Pod startup. You can use the `fsGroupChangePolicy` field inside a `securityContext`
+to control the way that Kubernetes checks and manages ownership and permissions
+for a volume.
+-->
+默认情况下，Kubernetes 在挂载一个卷时，会递归地更改每个卷中的内容的属主和访问权限，使之与 Pod
+的 `securityContext` 中指定的 `fsGroup` 匹配。
+对于较大的数据卷，检查和变更属主与访问权限可能会花费很长时间，降低 Pod 启动速度。
+你可以在 `securityContext` 中使用 `fsGroupChangePolicy` 字段来控制 Kubernetes
+检查和管理卷属主和访问权限的方式。
+
+<!--
+**fsGroupChangePolicy** -  `fsGroupChangePolicy` defines behavior for changing ownership and permission of the volume
+before being exposed inside a Pod. This field only applies to volume types that support
+`fsGroup` controlled ownership and permissions. This field has two possible values:
+
+* _OnRootMismatch_: Only change permissions and ownership if permission and ownership of root directory does not match with expected permissions of the volume. This could help shorten the time it takes to change ownership and permission of a volume.
+* _Always_: Always change permission and ownership of the volume when volume is mounted.
+
+For example:
+-->
+**fsGroupChangePolicy** - `fsGroupChangePolicy` 定义在卷被暴露给 Pod 内部之前对其
+内容的属主和访问许可进行变更的行为。此字段仅适用于那些支持使用 `fsGroup` 来
+控制属主与访问权限的卷类型。此字段的取值可以是：
+
+* `OnRootMismatch`：只有根目录的属主与访问权限与卷所期望的权限不一致时，
+  才改变其中内容的属主和访问权限。这一设置有助于缩短更改卷的属主与访问
+  权限所需要的时间。
+* `Always`：在挂载卷时总是更改卷中内容的属主和访问权限。
+
+例如：
+
+```yaml
+securityContext:
+  runAsUser: 1000
+  runAsGroup: 3000
+  fsGroup: 2000
+  fsGroupChangePolicy: "OnRootMismatch"
+```
+
+<!--
+This is an alpha feature. To use it, enable the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) `ConfigurableFSGroupPolicy` for the kube-api-server, the kube-controller-manager, and for the kubelet.
+-->
+这是一个 Alpha 阶段的功能特性。要使用此特性，需要在 `kube-apiserver`、`kube-controller-manager`
+和 `kubelet` 上启用 `ConfigurableFSGroupPolicy`
+[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)。
+
+<!--
+This field has no effect on ephemeral volume types such as
+[`secret`](/docs/concepts/storage/volumes/#secret),
+[`configMap`](/docs/concepts/storage/volumes/#configmap),
+and [`emptydir`](/docs/concepts/storage/volumes/#emptydir).
+-->
+{{< note >}}
+此字段对于[`Secret`](/zh/docs/concepts/storage/volumes/#secret)、
```

Comment:
```suggestion
此字段对于[`secret`](/zh/docs/concepts/storage/volumes/#secret)、
```

Reply:
correct, the volume type is `secret` rather than `Secret`.